docs/development/getting_started.rst

```diff
@@ -17,7 +17,7 @@ your development workstation.
 Ubuntu/Debian
 ~~~~~~~~~~~~~
 
-.. note:: Tested on: Ubuntu 14.04
+.. note:: Tested on: Ubuntu 14.04 and Debian sid (May 6th)
```

Comment:
I say we go with `Debian Stretch` because it's a more stable point of reference and @conorsch is running it, so if it breaks we'll know. Just replaced it with that.